tutorial/about-tutorial.md

```diff
@@ -91,9 +91,9 @@ Conversational procedural structure, with a few ordered steps described in the b
 No outcome statement.
 ```
 
-You get a more conversational flow with this tutorial style, however some of the prescriptive clarity is lost by not having the headings.
+You get a more conversational flow with this tutorial style, however, some of the prescriptive clarity is lost by not having the headings.
```

Comment:
Suggest splitting into two sentences:

You get a more conversational flow with this tutorial style. 
However, some of the prescriptive clarity is lost by not having the headings.

Reply:
Guess I'll go with the first suggestion.

Reply:
Yep the semicolon approach is fine here too.